website/docs/5.0-upgrade-guide.html.markdown

```diff
@@ -120,6 +120,10 @@ Please follow the format in the example below for listing breaking changes in re
 * The deprecated `storage_account_name` property has been removed in favour of the `storage_account_id` property.
 * The deprecated `resource_manager_id` property has been removed in favour of the `id` property.
 
+### `azurerm_express_route_connection`
+
+* The deprecated `private_link_fast_path_enabled` property has been removed as it is no longer supported by the resource.
+
```

Comment:
could this be moved further up to be in alphabetical order with the rest of the resources?

Reply:
updated